holmes/core/toolset_manager.py

```
@@ -0,0 +1,314 @@
+                isinstance(toolset_config["enabled"], bool)
+                and not toolset_config["enabled"]
+            ):
+                return False
+            # Normally benedict should translate the value of enabled to bool when well assigned,
+            # but in case it doesn't, and to avoid unexpected behavior, we check if the value is a
+            # string and if it is 'false' (case insensitive).
+            elif (
+                isinstance(toolset_config["enabled"], str)
+                and toolset_config["enabled"].lower() == "false"
+            ):
+                return False
```

Ruff (0.11.9)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)